financial-services-accelerator/components/org.wso2.financial.services.accelerator.common/src/main/java/org/wso2/financial/services/accelerator/common/extension/model/ExternalServiceRequest.java

```diff
@@ -27,6 +27,10 @@ public class ExternalServiceRequest {
     Event event;
     AllowedOperation allowedOperation;
 
+    public ExternalServiceRequest() {
```

Comment:
Do we have a usage of this?

Reply:
not yet

Reply:
Shall we remove it for now? It can be added when there is a usage.